shared/src/transport/http.rs

```diff
@@ -191,53 +206,23 @@ fn method_name(call: &Call) -> &str {
     }
 }
 
+#[derive(prometheus_metric_storage::MetricStorage, Clone, Debug)]
+#[metric(subsystem = "node_transport")]
 struct TransportMetrics {
+    /// Number of inflight RPC requests for ethereum node.
+    #[metric(labels("method"))]
```

Comment:
I couldn't find documentation for the MetricStorage derive in https://docs.rs/prometheus-metric-storage/0.2.0/prometheus_metric_storage/ and the metric attribute. Would be nice to have that like how other crates document macros.

Reply:
Yeah, documentation is in progress.